package.py

```diff
@@ -47,23 +47,24 @@
         "BoogieCodeContractsExtender",
         "BoogieConcurrency",
         "BoogieCore",
-        "DafnyPipeline",
-        "Dafny",
         "BoogieDoomed",
         "BoogieExecutionEngine",
         "BoogieGraph",
         "BoogieHoudini",
         "BoogieModel",
         "BoogieModelViewer",
         "BoogieParserHelper",
-        "Provers.SMTLib",
         "BoogieVCExpr",
         "BoogieVCGeneration",
+        "Dafny",
+        "DafnyRuntime",
+        "DafnyPipeline",
         "Mono.Cecil",
+        "Provers.SMTLib",
         "System.Collections.Immutable",
         "System.Runtime"]
 EXEs = ["Dafny", "DafnyServer"]
-ETCs = UNIX_EXECUTABLES + ["DafnyPrelude.bpl", "DafnyRuntime.cs", "DafnyRuntime.js", "DafnyRuntime.go", "DafnyRuntime.jar", "libhostpolicy.dylib"]
+ETCs = UNIX_EXECUTABLES + ["DafnyPrelude.bpl", "DafnyRuntime.cs", "DafnyRuntime.js", "DafnyRuntime.go", "DafnyRuntime.jar", "Dafny.deps.json", "Dafny.runtimeconfig.json"]
```

Comment:
Is `package.py` meant to be part of this PR?

Reply:
No - fixed it